From 42d15fb1393b791af9e279027a852620be3d0336 Mon Sep 17 00:00:00 2001
From: Lubomir Rintel <lkundrak@v3.sk>
Date: Sat, 29 Oct 2016 15:39:23 +0200
Subject: [PATCH] Load qtparts on build

This ensures the proper CXXFLAGS for a particular Linux distro would be
used.
---
 qtl866.pro | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/qtl866.pro b/qtl866.pro
index ba2965f..84589bb 100644
--- a/qtl866.pro
+++ b/qtl866.pro
@@ -6,7 +6,10 @@
 
 QT       += core gui
 
-greaterThan(QT_MAJOR_VERSION, 4): QT += widgets
+greaterThan(QT_MAJOR_VERSION, 4): {
+    QT += widgets
+    load(qt_parts)
+}
 
 TARGET = qtl866
 TEMPLATE = app
